### PR TITLE
Create MP7 zip

### PR DESCRIPTION
**Pull request by tevans78**
Create the MicroProfile7 OL zip

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
PB is good. MP 7 zip is here - https://libfsfe05.hursley.ibm.com/liberty/personal/1/tevans78-28686-20240610-0709-_ZhSNICcpEe-8orPX4FHM1A/fe/tevans78-28686-20240610-0709.28.linux/linux/zipper/externals/installables/openliberty-microProfile7-24.0.0.7-202406101018.zip

File: dev/build.image/build.gradle [hunk 1]
```diff
@@ -636,6 +637,21 @@ if (isAutomatedBuild && !isIFIXBuild) {
             }
         }
     }
+    
+    //currently noship
+    task packageOpenLibertyMicroProfile7(type: PackageLibertyWithFeatures) {
+        dependsOn parent.subprojects.assemble
+        dependsOn ':com.ibm.websphere.appserver.features:publishFeatureResources'
+        withFeatures this.&microProfile7Features
+        packageServerConflict = "io.openliberty.microProfile,io.openliberty.mpConfig,io.openliberty.mpTelemetry,io.openliberty.org.eclipse.microprofile.config,io.openliberty.mpCompatible,io.openliberty.internal.mpVersion"
```
Comment (Azquelt) [inline dev/build.image/build.gradle:646]: We think there shouldn't be this many features here. MP7, and therefore the MP7 zip, shouldn't have multiple versions of MP features in it.

**Review by Azquelt — APPROVED** [review]

File: dev/build.image/build.gradle (hunk 1 above)
Comment (tevans78) [reply dev/build.image/build.gradle:646]: will address in a follow up

**Comment by tevans78** [comment]
#libby

**Pull request merged**